migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/migration.go

```diff
@@ -251,6 +255,52 @@ func migrateRoles(gormDB *gorm.DB, postgresDB *pgxpool.Pool, legacyStore legacyr
 	return nil
 }
 
+func getReportConfigurationScopeID(reportConfiguration *storage.ReportConfiguration) (string, error) {
+	reportConfigurationScopeID := strings.TrimPrefix(reportConfiguration.GetScopeId(), accessScopeIDPrefix)
+	if replacement, found := accessScopeIDMapping[reportConfigurationScopeID]; found {
+		reportConfigurationScopeID = replacement
+	}
+	_, accessIDParseErr := uuid.FromString(reportConfigurationScopeID)
+	if accessIDParseErr != nil {
+		log.WriteToStderrf("failed to convert report configuration to postgres format, bad scope ID. Report Configuration [%s], error %v", reportConfiguration.GetName(), accessIDParseErr)
+		return "", accessIDParseErr
+	}
+	return reportConfigurationScopeID, nil
+}
+
+func updateReportConfigurationScopeIDs(postgresDB *pgxpool.Pool) error {
+	ctx := sac.WithAllAccess(context.Background())
+	store := pgReportConfigurationStore.New(postgresDB)
+	var reportConfigs []*storage.ReportConfiguration
+	err := walkReportConfigurations(ctx, store, func(obj *storage.ReportConfiguration) error {
+		reportConfigurationScopeID, conversionErr := getReportConfigurationScopeID(obj)
+		if conversionErr != nil {
+			return conversionErr
+		}
```

Comment:
Do we want to fail open vs fail closed here?
i.e. do we want to abort migration of _all_ report configuration because we have a single entry with an unexpected ID, or do we rather want to log the occurrence of it, and continue with migrating the other report configurations?

I know we have had a mix of those concepts in the rocksdb migrations, not sure how we have converged on the postgres migrations or its still more or less a case-by-case basis.